src/rbus/rbus.c

```diff
@@ -4257,7 +4301,7 @@ rbusError_t rbusEvent_Unsubscribe(
 
     /*the use of rtVector is inefficient here.  I have to loop through the vector to find the sub by name, 
         then call RemoveItem, which loops through again to find the item by address to destroy */
-    sub = rbusEventSubscription_find(handleInfo->eventSubs, eventName, NULL);
+    sub = rbusEventSubscription_find(handleInfo->eventSubs, eventName, NULL, NULL, NULL);
```

Comment:
Pass, filter as NULL(already done), interval as 0 and duration as 0.

Reply:
done